### DISCUSSION TITLE
[RFC] Use Bolt Collections instead of Laravel ones

### DISCUSSION THREAD

**Discussion by JarJak** [bolt/core#325] [Ideas / Feature requests]
@bobdenotter why we don't use https://github.com/bolt/collection ? It looks great, contains immutability and can be adapted to PHP7 in 2.0 version.

**Comment by arnofo** [comment]
Have you considered using https://github.com/doctrine/collections ? Or is it missing a feature you need ?

Since you already use Doctrine this would mean one less dependency.

**Comment by bobdenotter** [comment]
I personally think we should stop using bolt/collections.. Not "re-inventing the  wheel", basically, having one less thing to maintain ourselves.

And, it's a matter of personal preference certainly, but i like Laravel's collections. It has much more methods than Doctrine's. :-)
